Cargo.toml

```diff
@@ -6,7 +6,9 @@ version = "0.14.0"
 authors = [
     "Alex Morega <alex@grep.ro>",
     "Johannes Drönner <droenner@informatik.uni-marburg.de>",
+    "GeoRust GDAL Team"
```

Comment:
I'd remove this instead, https://rust-lang.github.io/rfcs/3052-optional-authors-field.html.

Reply:
@lnicola I wouldn't want to offend Alex or Johannes. I'll drop this addition. Again, you caught me doing PR scope creep.